ci/taos/webapp/install-packages-nnstreamer.sh

```diff
@@ -52,5 +52,6 @@ echo -e "########## for CI-system: Installing python packages for nnstreaemr app
 sudo apt -y install python-gi python3-gi  
 sudo apt -y install python-gst-1.0 python3-gst-1.0
 sudo apt -y install python-gst-1.0-dbg python3-gst-1.0-dbg
+sudo apt -y install python-numpy python3-numpy
```

Comment:
Please check it again. 
[FYI] I have checked the package name both Python2 and Python3.
```bash
invain@u1604:~$ dpkg -S /usr/bin/python
python-minimal: /usr/bin/python
invain@u1604:~$ dpkg -S /usr/bin/python2
python-minimal: /usr/bin/python2
invain@u1604:~$ dpkg -S /usr/bin/python3
python3-minimal: /usr/bin/python3
invain@u1604:~$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"

```

Reply:
Please, Ignore my comment. The change will be normal in the real server.

Reply:
I have done the statement in the nnusite.mooo.com server.